examples/graphiql-create-react-app/public/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
did create react app generate this file?

Reply:
Yes it was generated by CRA, I modified the meta info a bit to suit the context.